### ISSUE TITLE
ST3 Build 3119 no Syntax Highlighting Whatsoever

### ISSUE THREAD

**Issue by bullitbd** [jonschlinkert/sublime-markdown-extended#137]
* Installed Monokai Extended
* Installed Markdown Extended

there is no markdown styling applied; Interestingly, Markdown Extended is not included in the package list:

![screen shot 2016-08-02 at 2 32 06 pm](https://cloud.githubusercontent.com/assets/9753407/17346307/d9a4bf54-58be-11e6-867c-9364fc44b5cb.png)

![screen shot 2016-08-02 at 2 37 15 pm](https://cloud.githubusercontent.com/assets/9753407/17346318/e71fb99a-58be-11e6-8a47-8fcab0b9a328.png)

![screen shot 2016-08-02 at 2 37 39 pm](https://cloud.githubusercontent.com/assets/9753407/17346326/f11145a4-58be-11e6-8cdc-e7bd8619482b.png)

**Comment by doowb** [comment]
> Interestingly, Markdown Extended is not included in the package list

Does it show up in the `Preferences > Color Scheme`?

**Comment by bullitbd** [comment]
_Monokai Extended_ shows up there...

**Comment by jonschlinkert** [comment]
Hmm, strange. I know this is totally obvious, but did you try restarting ST? It's also possible that another library is conflicting

**Comment by bullitbd** [comment]
yes, unfortunately...restarted several times of course - uninstalled, cleaned caches, reinstalled, etc.

 anything else I can send along, or process I might follow for you??

**Comment by jonschlinkert** [comment]
> yes, unfortunately

I figured... thanks for the additional info. Have you played around with the settings in `Preferences > Color Scheme > Monokai Extended`? If none of the themes are working that seems like a sign of a plugin conflict

> anything else I can send along, or process I might follow for you??

I wish I knew more about ST internals so I could give you some guidance there. One thing you could try is uninstalling this package, then make sure to restart ST. Then do the same with other packages and try reinstalling this to see if it's conflicting with another plugin. I know that's a pain but it might help narrow down the cause.

**Comment by bullitbd** [comment]
I did that, a couple of times - also tried MarkdownEditing, which works properly... and disabled a number of packages... when I get a chance, I'll spool up an empty ST instance and try there. Will report - thanks.

**Comment by jonschlinkert** [comment]
I really appreciate it, sorry it's not working for you I hope we get this figured out

**Comment by yangnw** [comment]
same here, clean ST3 install, no effect at all. doesn't show under preferences. MarkdownEditing works though.

**Comment by jonschlinkert** [comment]
wait, I just realized that the example shows you looking in `Package Settings`. The readme explains how to enable this: https://github.com/jonschlinkert/sublime-markdown-extended#2-activate-this-language

**Comment by solo12zw74** [comment]
The same for me in build 3114. Open console and saw errors:

> Error loading scope:source.css.less: Unable to find syntax file for scope source.css.less in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:text.html.handlebars: Unable to find syntax file for scope text.html.handlebars in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.sass: Unable to find syntax file for scope source.sass in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.nginx: Unable to find syntax file for scope source.nginx in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.ini: Unable to find syntax file for scope source.ini in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.swift: Unable to find syntax file for scope source.swift in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.livescript: Unable to find syntax file for scope source.livescript in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.jade: Unable to find syntax file for scope source.jade in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.ejs: Unable to find syntax file for scope source.ejs in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.julia: Unable to find syntax file for scope source.julia in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.coffee: Unable to find syntax file for scope source.coffee in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.scss: Unable to find syntax file for scope source.scss in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.stylus: Unable to find syntax file for scope source.stylus in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.pgsql: Unable to find syntax file for scope source.pgsql in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:text.html.swig: Unable to find syntax file for scope text.html.swig in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> Error loading scope:source.postscript: Unable to find syntax file for scope source.postscript in Packages/Markdown Extended/Syntaxes/Markdown Extended.sublime-syntax
> rule scope:source.rust##anon_struct-tuple_0 has a scope name, but is unreachable, so the name will never be used
> rule #anon_latex_0 has a scope name, but is unreachable, so the name will never be used
> rule #anon_latex_1 has a scope name, but is unreachable, so the name will never be used

PS plugin installed with Package Control

**Comment by ssteinerx** [comment]
Late to the party, Sublime Dev Channel, Build 3132, same symptoms: not in list of available syntax hilighters, nor is it in the available "Package Settings".

This is a completely stripped down installation with only Package Control, markdown extended and monokai extended so I'm not sure what could be going on.  I will definitely post back if I figure it out...

**Comment by ssteinerx** [comment]
Well...that was quick.  In my case the problem was that the `~/Library/Application Support/Sublime Text 3/User` symlink had gone bad during a disk shuffle having *nothing to do with Sublime*.

When I re-established the link, there were a ton of stray settings etc. so I started over from scratch by removing *all* installed packages except for Package Control, moving the contents of the old directory aside, and starting over from scratch, moving settings back in as I installed each package.

Seems pretty drastic, but it really only took about 1/2 hour, Sublime loads faster, runs more smoothly, and some niggling issues with the sidebar went away.  Plus, all that fung from old, uninstalled packages etc. are gone.

Not necessarily super-optimal, time-wise, but 100% effective!

**Comment by afeique** [comment]
I'm getting the same problem running ST3 x64 build 3126 on Windows 7 x64.

Both Monokai Extended and Markdown Extended were installed (in that order).

Verified that the plugin installed okay using Package Controller and appears under `\\AppData\Roaming\Sublime Text 3\Installed Packages\Markdown Extended.sublime-package`. No errors in console during/after installation.

**Monokai Extended activated**
![image](https://user-images.githubusercontent.com/766889/27925575-47fdb192-6253-11e7-91bb-fa3072d61ec7.png)

**Syntax = Markdown Extended**
![image](https://user-images.githubusercontent.com/766889/27925617-78d74aee-6253-11e7-8589-53af6ec8dc70.png)

**Comment by laukstein** [comment]
Same error on ST3 build 3176
